src/core/api/index.js

```diff
@@ -111,7 +111,11 @@ export function search({ api, page, auth = false, filters = {} }) {
     endpoint: 'addons/search',
     schema: { results: [addon] },
     params: {
-      app: api.clientApp,
+      // We allow overrides to clientApp because of
+      // https://github.com/mozilla/addons-frontend/issues/1723
+      // When it stops requiring overrides to the clientApp this
+      // can likely be restored to `api.clientApp` only.
+      app: filters.clientApp || api.clientApp,
```

Comment:
It's here: https://github.com/mozilla/addons-frontend/pull/1875/files#diff-da8c6fe27817853c3f04c9897c485535R247

But it's implicit because it's also about switching android -> firefox. I'll make it clearer.